src/styles/block/cards.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px;
+  margin-bottom: 48px;
+
+  &__image {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 160px;
+    height: 134px;
+    margin: 32px auto 40px;
+  }
+
+  &__text {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    font-weight: 400;
```

Comment:
It's the default value
![image](https://user-images.githubusercontent.com/91411692/222675732-2c874b59-93fa-4eae-9add-f57c0ce9ebcb.png)

![image](https://user-images.githubusercontent.com/91411692/222675648-684def54-5cf1-4c04-b707-21d0503a2dfb.png)

```suggestion
```